bitnami/mariadb-galera/templates/prometheusrules.yaml

```diff
@@ -9,14 +9,10 @@ kind: PrometheusRule
 metadata:
   name: {{ template "common.names.fullname" . }}
   namespace: {{ include "common.names.namespace" . | quote }}
-  labels: {{- include "common.labels.standard" . | nindent 4 }}
-  {{- if .Values.commonLabels }}
-  {{- include "common.tplvalues.render" (dict "value" .Values.commonLabels "context" $) | nindent 4 }}
-  {{- end }}
-  {{- include "common.tplvalues.render" ( dict "value" .Values.metrics.prometheusRules.additionalLabels "context" $ ) | nindent 4 }}
-  {{- if .Values.commonAnnotations }}
-  annotations: {{- include "common.tplvalues.render" ( dict "value" .Values.commonAnnotations "context" $ ) | nindent 4 }}
-  {{- end }}
+  labels: {{- include "common.labels.standard" ( dict "customLabels" .Values.commonLabels "context" $ ) | nindent 4 }}
+    {{- if .Values.metrics.prometheusRule.additionalLabels }}
+    {{- include "common.tplvalues.render" ( dict "value" .Values.metrics.prometheusRules.additionalLabels "context" $ ) | nindent 4 }}
+    {{- end }}
```

Comment:
I think it is better to follow the same approach used in other templates where you declare a variable and merge all the labels together

Reply:
I'm only doing so when there's a `.Values.XXX.labels` parameter and skipping it when the parameter name is `.Values.XXX.additionalLabels` since the naming suggests they should be added (not merged)